### PR TITLE
New draggables look

### PR DESCRIPTION
**Pull request by vxsx**
Before:
screen shot 2015-09-03 at 16 58 23

After:
screen shot 2015-09-03 at 16 57 49

### CODE REVIEW

File: cms/static/cms/sass/settings/_cms.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $speed-base: 200ms;
 $white: #fff;
 $black: #000;
 
-$color-primary: #57bcf0;
+$color-primary: #52bdf3;
```
Comment (Brankub) [inline cms/static/cms/sass/settings/_cms.scss]: :trophy:  :-D

File: cms/templates/cms/toolbar/dragitem.html [hunk 2]
```diff
@@ -4,6 +4,12 @@
 <div class="cms-draggable cms-draggable-{{ plugin.pk|unlocalize }}{% if plugin.disable_child_plugins %} cms-draggable-disabled{% endif %}">
     <div class="cms-dragitem cms-dragitem-handler{% if plugin.child_plugin_instances %} cms-dragitem-collapsable{% endif %}">
         {% language request.toolbar.toolbar_language %}
+        {% if disabled_child %}
+        <span class="cms-hover-tooltip cms-hover-tooltip-left cms-plugin-disabled" tabindex="-1"
```
Comment (Brankub) [inline cms/templates/cms/toolbar/dragitem.html]: please fix indentation

File: cms/static/cms/sass/settings/_cms.scss (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/sass/settings/_cms.scss]: props to Mathias!

Comment (Brankub) [inline cms/static/cms/sass/settings/_cms.scss]: :+1:

**Comment by Brankub** [comment]
LGTM!

**Pull request merged**